### PR TITLE
Readme draft

### PR DESCRIPTION
**Pull request by csralvall**
This pull request creates a first draft of the README's spanish translation from the master branch.

I tried to keep all the link names equal to their file names to avoid confusion.

All suggestions and changes are welcome.

### CODE REVIEW

File: es/LEEME.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+### PARTE DOS: USANDO BITCOIN-CLI
+
+**Estado:** Finalizado. Actualizado para la versión 0.20.
+
+* [3.0: Entendiendo Su Configuración Bitcoin](03_0_Entendiendo_Su_Configuracion_Bitcoin.md)
+  * [3.1: Verificando Su Configuración Bitcoin](03_1_Verificando_Su_Configuracion_Bitcoin.md)
+  * [3.2: Conociendo Su Configuración Bitcoin](03_2_Conociendo_Su_Configuracion_Bitcoin.md)
+  * [3.3: Configurando Su Billetera](03_3_Configurando_Su_Billetera.md)
+    * [Interludio: Usando Variables de la Interfaz de Linea de Comandos](03_3__Interludio_Usando_Variables_de_la_Interfaz_de_Linea_de_Comandos.md)
```
Comment (javiervargas) [inline es/LEEME.md:32]: 03_3_Interludio_Usando_Variables_Linea_Comando.md

File: es/LEEME.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+  * [3.4: Recibiendo una Transacción](03_4_Recibiendo_una_Transaccion.md)
+  * [3.5: Entendiendo el Descriptor](03_5_Entendiendo_el_Descriptor.md)
```
Comment (javiervargas) [inline es/LEEME.md:34]: 03_5_Entendiendo_El_Descriptor.md

File: es/LEEME.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+  * [5.3: Financiando una Transacción con CPFP](05_3_Financiando_una_Transaccion_con_CPFP.md)
+* [6.0: Expandiendo las Transacciones Bitcoin con Multifirmas](06_0_Expandiendo_las_Transacciones_Bitcoin_con_Multifirmas.md)
+  * [6.1: Enviando una Transacción a una Dirección Multifirma](06_1_Enviando_una_Transaccion_a_una_Direccion_Multifirma.md)
+  * [6.2: Gastando una Transacción con una Dirección Multifirma](06_2_Gastando_una_Transaccion_con_una_Direccion_Multifirma.md)
+  * [6.3: Enviando y Gastando una Multifirma Automatizada](06_3_Enviando_una_Multifirma_Automatizada.md)
+* [7.0: Expandiendo las Transacciones Bitcoin con PSBTs](07_0_Expandiendo_las_Transacciones_Bitcoin_con_PSBTs.md)
+  * [7.1: Creando una Transacción Bitcoin Parcialmente Firmada](07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md)
+  * [7.2: Usando una Transacción Bitcoin Parcialmente Firmada](07_2_Usando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md)
+  * [7.3: Integrando con Hardware Wallets](07_3_Integrando_con_Hardware_Wallets.md)
+* [8.0: Expandiendo Transacciones Bitcoin en Otros Sentidos](08_0_Expandiendo_Transacciones_Bitcoin_en_Otros_Sentidos.md)
```
Comment (javiervargas) [inline es/LEEME.md:56]: 08_0_Expandiendo_Bitcoin_Transacciones_Otros.md

File: es/LEEME.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+* [6.0: Expandiendo las Transacciones Bitcoin con Multifirmas](06_0_Expandiendo_las_Transacciones_Bitcoin_con_Multifirmas.md)
+  * [6.1: Enviando una Transacción a una Dirección Multifirma](06_1_Enviando_una_Transaccion_a_una_Direccion_Multifirma.md)
+  * [6.2: Gastando una Transacción con una Dirección Multifirma](06_2_Gastando_una_Transaccion_con_una_Direccion_Multifirma.md)
+  * [6.3: Enviando y Gastando una Multifirma Automatizada](06_3_Enviando_una_Multifirma_Automatizada.md)
+* [7.0: Expandiendo las Transacciones Bitcoin con PSBTs](07_0_Expandiendo_las_Transacciones_Bitcoin_con_PSBTs.md)
+  * [7.1: Creando una Transacción Bitcoin Parcialmente Firmada](07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md)
+  * [7.2: Usando una Transacción Bitcoin Parcialmente Firmada](07_2_Usando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md)
+  * [7.3: Integrando con Hardware Wallets](07_3_Integrando_con_Hardware_Wallets.md)
+* [8.0: Expandiendo Transacciones Bitcoin en Otros Sentidos](08_0_Expandiendo_Transacciones_Bitcoin_en_Otros_Sentidos.md)  
+  * [8.1: Enviando una Transacción con un Bloqueo de Tiempo](08_1_Enviando_una_Transaccion_con_un_Bloqueo_de_Tiempo.md)
```
Comment (javiervargas) [inline es/LEEME.md:57]: 08_1_Enviando_una_Transaccion_con_Bloqueo_de_Tiempo.md

File: es/LEEME.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+  * [8.2: Enviando una Transacción con Datos](08_2_Enviando_una_Transaccion_con_Datos.md)
+  
+### PARTE TRES: BITCOIN SCRIPTING
+
+**Estado:** Finalizado. Actualizado para 0.20 y btcdeb.
+
+* [9.0: Introduciendo Bitcoin Scripts](09_0_Introduciendo_Bitcoin_Scripts.md)
+  * [9.1: Entendiendo la Base de las Transacciones](09_1_Entendiendo_la_Base_de_las_Transacciones.md)
+  * [9.2: Ejecutando un Bitcoin Script](09_2_Ejecutando_un_Bitcoin_Script.md)
```
Comment (javiervargas) [inline es/LEEME.md:66]: 09_2_Ejecutando_un_Script_Bitcoin.md

File: es/LEEME.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+  * [9.3: Testeando un Bitcoin Script](09_3_Testeando_un_Bitcoin_Script.md)
```
Comment (javiervargas) [inline es/LEEME.md:67]: 09_3_Probando_un_Script_Bitcoin.md

File: es/LEEME.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+  * [9.4: Programando una P2PKH](09_4_Programando_una_P2PKH.md)
```
Comment (javiervargas) [inline es/LEEME.md:68]: 09_4_Codificando_una_P2PKH.md

File: es/LEEME.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+  * [9.5: Programando una P2WPKH](09_5_Programando_una_P2WPKH.md)
```
Comment (javiervargas) [inline es/LEEME.md:69]: 09_5_Codificando_una_P2WPKH.md

File: es/LEEME.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+  * [10.2: Construyendo la Estructura de P2SH](10_2_Construyendo_la_Estructura_de_P2SH.md)
+  * [10.3: Ejecutando un Script Bitcoin con P2SH](10_3_Ejecutando_un_Script_Bitcoin_con_P2SH.md)
+  * [10.4: Programando una Multifirma](10_4_Programando_una_Multifirma.md)
+  * [10.5: Programando un Script Segwit](10_5_Programando_un_Script_Segwit.md)
+  * [10.6: Gastando una Transacción P2SH](10_6_Gastando_una_Transaccion_P2SH.md)
+* [11.0: Potenciando Bloqueos de Tiempo con Bitcoin Scripts](11_0_Potenciando_Bloqueos_de_Tiempo_con_Bitcoin_Scripts.md)
+  * [11.1: Entendiendo las Opciones de los Bloqueos de Tiempo](11_1_Entendiendo_las_Opciones_de_los_Bloqueos_de_Tiempo.md)
+  * [11.2: Usando CLTV en Scripts](11_2_Usando_CLTV_en_Scripts.md)
+  * [11.3: Usando CSV en Scripts](11_3_Usando_CSV_en_Scripts.md)
+* [12.0: Expandiendo Bitcoin Scripts](12_0_Expandiendo_Bitcoin_Scripts.md)
```
Comment (javiervargas) [inline es/LEEME.md:81]: 12_0_Expandiendo_Scripts_Bitcoin.md

File: es/LEEME.md [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+  * [12.1: Usando Condicionales](12_1_Usando_Condicionales.md)
```
Comment (javiervargas) [inline es/LEEME.md:82]: 12_1_Usando_Script_Condicionales.md

**Review by javiervargas — CHANGES_REQUESTED** [review]
I have proposed some changes and I have included others that are already codified in several chapters

File: es/LEEME.md (hunk 1 above)
Comment (csralvall) [reply es/LEEME.md:32]: Should I change also the link name?

Comment (csralvall) [reply es/LEEME.md:32]: Should I also change the link names (inside [])?

File: es/LEEME.md (hunk 3 above)
Comment (csralvall) [reply es/LEEME.md]: How the link name should be?

File: es/LEEME.md (hunk 1 above)
Comment (javiervargas) [reply es/LEEME.md]: Yes!, sorry.

**Review by javiervargas — APPROVED** [review]

File: es/LEEME.md (hunk 3 above)
Comment (csralvall) [reply es/LEEME.md]: I will kept the link name, because it's a literal translation from the english one.

File: es/LEEME.md (hunk 1 above)
Comment (csralvall) [reply es/LEEME.md]: It's done. I added some articles to make it not so cryptic and more readable, I hope it's ok.

**Pull request closed without merging**